# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsParent.cpp
# Commit: 498d5e653e97
# Full Hash: 498d5e653e9708d153749ad3b71349fa30f69e00
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-01-06 21:54:03
# Regressor Bug: 1600906
# File Overlap Count: 1
# Description:
#   Bug 1600906 - Avoid unnecessary copy. r=dom-workers-and-storage-reviewers,ttung
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D57986
# ==============================================================================

diff -r 90cec6ee12ac -r 498d5e653e97 dom/indexedDB/ActorsParent.cpp
--- a/dom/indexedDB/ActorsParent.cpp	Mon Jan 06 09:52:30 2020 +0000
+++ b/dom/indexedDB/ActorsParent.cpp	Fri Dec 20 15:32:36 2019 +0000
@@ -14167,7 +14167,7 @@
   if (objMetadata->mCommonMetadata.autoIncrement() &&
       objMetadata->mCommonMetadata.keyPath().IsValid() &&
       aParams.key().IsUnset()) {
-    const SerializedStructuredCloneWriteInfo cloneInfo = aParams.cloneInfo();
+    const SerializedStructuredCloneWriteInfo& cloneInfo = aParams.cloneInfo();
 
     if (NS_WARN_IF(!cloneInfo.offsetToKeyProp())) {
       ASSERT_UNLESS_FUZZING();
